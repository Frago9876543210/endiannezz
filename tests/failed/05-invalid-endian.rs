use endiannezz::Io;

#[derive(Io)]
#[endian(invalid)]
struct Foo;

fn main() {}
