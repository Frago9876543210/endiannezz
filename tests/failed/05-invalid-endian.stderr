error: failed to determine endian
 --> $DIR/05-invalid-endian.rs:4:10
  |
4 | #[endian(invalid)]
  |          ^^^^^^^
